lib/src/controllers/player_controller.dart

```diff
@@ -250,7 +250,7 @@ class PlayerController extends ChangeNotifier {
   /// otherwise nothing happens.
   Future<void> seekTo(int progress) async {
     if (progress < 0) return;
-    if (_playerState == PlayerState.playing) {
+    if (_playerState == PlayerState.playing || _playerState == PlayerState.paused || _playerState == PlayerState.initialized) {
```

Comment:
this could be simplified like this,
```suggestion
    if (progress < 0 || _playerState.isStopped) return;
```